src/features/client/components/Discovery/index.jsx

```diff
@@ -0,0 +1,54 @@
+import { Box, Heading, Flex, Button } from '@chakra-ui/react';
+import { ChevronLeftIcon, ChevronRightIcon } from '@chakra-ui/icons';
+import { restaurantInspectMock } from '../../mocks/restaurantInspectMock';
+import './index.scss';
+import RestaurantInspectForLanding from '../RestaurantInspectForLanding';
+
+function Discovery() {
+  return (
+    <Box as="section" className="container" position="relative">
+      <Heading>Discover</Heading>
+      <Flex alignItems="center" mt={5}>
+        <Button
+          position="absolute"
```

Comment:
try to provide properties for position (top, bottom, right, left (whichever you want)), could not look the same on other browsers without them.